### PR TITLE
Update about-shared-mailboxes.md

### PR DESCRIPTION
**Pull request by guruxp**
added a change on the Too Many Users section so we state no more than 25 users is recommended per shared mailbox concurrently. This will help customers better plan their needs, avoid unnecessary cases from being opened and have an article we can answer from support regarding what shared mailboxes are for in terms of sizing.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/email/about-shared-mailboxes.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Before you [create a shared mailbox](create-a-shared-mailbox.md), here are some
 
 - **Signing in:** A shared mailbox is not intended for direct sign-in by its associated user account. You should always block sign-in for the shared mailbox account and keep it blocked.
 
-- **Too many users:** When there are too many designated users concurrently accessing a shared mailbox, they may intermittently fail to connect to this mailbox. In this case, you can consider reducing the number of the users or using a different workload, such a Microsoft 365 group or Public folder.
+- **Too many users:** When there are too many designated users concurrently accessing a shared mailbox (no more than 25 is recommended), they may intermittently fail to connect to this mailbox or have inconsistencies like messages being duplicate on the outbox. In this case, you can consider reducing the number of the users or using a different workload, such a Microsoft 365 group or Public folder.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/about-shared-mailboxes.md]:
```suggestion
- **Too many users:** When there are too many designated users concurrently accessing a shared mailbox (no more than 25 is recommended), they may intermittently fail to connect to this mailbox or have inconsistencies like messages being duplicated in the outbox. In this case, you can consider reducing the number of users or using a different workload, such as a Microsoft 365 group or a Public folder.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @guruxp, kindly commit the suggested changes. Many thanks.

**Comment by guruxp** [comment]
@JohanFreelancer9 done :) thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by guruxp** [comment]
@JohanFreelancer9 do you know when this will go public?

**Pull request merged**